release/models/interfaces/openconfig-if-ip.yang

```diff
@@ -605,6 +620,16 @@ revision "2023-06-30" {
         "RFC 4862: IPv6 Stateless Address Autoconfiguration";
     }
 
+    leaf unsolicited-na-accepted {
```

Comment:
Minor, but I'd propose to consider other names (I left a note in another comment below, `accepted` is not a completely  accurate way to represent the meaning)

Reply:
I've updated to `learn-unsolicited`